waiter/integration/waiter/token_request_test.clj

```diff
@@ -374,10 +374,10 @@
                 services (json/read-str body)
                 service-tokens (mapcat (fn [entry]
                                          (some->> entry
-                                                  walk/keywordize-keys
```

Comment:
Nitpick: Unrelated formatting changes throughout this file

Reply:
Undone.